Exporter/src/main/java/uk/ac/ebi/biostd/exporter/configuration/ExporterGeneralProperties.java

```diff
@@ -1,13 +1,17 @@
 package uk.ac.ebi.biostd.exporter.configuration;
 
 import java.util.List;
-import lombok.Data;
+import lombok.Setter;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.stereotype.Component;
 
-@Data
+@Setter
 @Component
 @ConfigurationProperties(prefix = "jobs")
 public class ExporterGeneralProperties {
     private List<String> libFileStudies;
+
+    public List<String> getLibFileStudies() {
```

Comment:
I think we should apply this changes in the source of the problem (Dao) not here